.travis.yml

```diff
@@ -18,6 +18,7 @@ cache:
     - $HOME/.cabal
     - $TRAVIS_BUILD_DIR/.cache
     - $TRAVIS_BUILD_DIR/node_modules
+    - $TRAVIS_BUILD_DIR/versions
```

Comment:
The benefit here would be "the amount of time it takes to untar the cache" - I intentionally did not cache this part - we might make all sorts of changes to an installed node version over the course of tests, and making tests deterministic is far more important than making them fast. Let's remove this commit.